tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "sourceMap": true,
     "composite": true,
     "incremental": true,
-    "exactOptionalPropertyTypes": true
+    "exactOptionalPropertyTypes": true,
+    "importsNotUsedAsValues": "error"
```

Comment:
what if we enforce this only in `packages/target-ethers-v5/tsconfig.json`? I don't feel that it's necessary to use accross the whole codebase.